tests/Unit/LocoTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
I see I used my default PhpStorm template here, which uses `strict_types`. If you don't want to use that in this project, let me know, and I will remove this.

Reply:
No problem! :+1: